src/main.cpp

```diff
@@ -1,9 +1,117 @@
 #include "cloud4rpi.h"
 
-Cloud4RPi::Cloud4RPi(const char *deviceToken, const char* server) {
+Cloud4RPi::Cloud4RPi(const char *deviceToken, Client *client, const char* server) {
     _deviceToken = deviceToken;
```

Comment:
I'm not sure the C++ code style allows the "_" symbol anywhere, moreover in the private member indication. It looks very Pythonic